qtpy/__init__.py

```diff
@@ -251,38 +274,39 @@ def get_api_information(api_name):
 
 for api_name in api_trial:
     try:
-        api_version, qt_version = get_api_information(api_name)
+        API_VERSION, QT_VERSION = get_api_information(api_name)
     except PythonQtError:
         pass
     else:
-        if api_version and qt_version:
-            API = api_name
-            API_VERSION = api_version
-            QT_VERSION = qt_version
-            if API == 'PyQt4':
+        if API_VERSION and QT_VERSION:
+            API = api_name.lower()
+            API_NAME = api_name
+            if api_name == 'PyQt4':
```

Comment:
Need to fix these values not in lowercase?

Reply:
hi @goanpeca , inside the code, I'm using the importing name because we can pass it to the functions to import. The lowercase is in API.

In the future, a better idea to implement the function get_api_information(api_name) is to use pgkutil and/or importlib. When you use try...import, all the modules that you have tried are in sys.modules. And both libs accept a string to make the temporary import. I didn't do this way now because I need to make sure of what implications this can make [here](https://github.com/spyder-ide/qtpy/pull/156/files#diff-bca24c6a427bc0a1c74aa972ec74d9f6R172)

Reply:
The CI problem requires changes in test also because the order for attempts (PyQt5, PySide2, etc..) have changed. I'm working on it.